src/components/helpers/helper.js

```diff
@@ -1,17 +1,19 @@
 import axios from 'axios';
 
-const youtubeClient = axios.create({
+export const youtubeClient = axios.create({
   baseURL: 'https://www.googleapis.com/youtube/v3/',
   params: {
     part: 'snippet',
-    key: process.env.REACT_APP_API_KEY,
+    maxResults: 12,
+    key: 'AIzaSyAYqG4dkr8FeUIXfKTTqqrroxJo0tQENGM',
```

Comment:
`REACT_APP_API_KEY` is the way to go as exposing this key in github makes it available for anyone to steal this key